docs/v2/features-overzicht.md

```diff
@@ -163,6 +163,168 @@ In het volgende feature bestand zijn de bovenstaande regels geïllustreerd aan d
 
 - [verblijfstitelvelden vragen met fields](./features/persoon/verblijfstitel/overzicht.feature)
 
+## Vragen om gezag over een minderjarige
+
+Je kan om het gezag over een minderjarige vragen met fields "gezag". Je kan hier niet vragen om alleen specifieke velden binnen gezag, zoals "gezag.minderjarige", je krijgt bij gezag alles of niets.
+
+Gezag kan alleen gevraagd worden bij de zoekoperaties RaadpleegMetBurgerservicenummer en ZoekMetAdresseerbaarObjectIdentificatie. Gezag vragen met ZoekMetAdresseerbaarObjectIdentificatie kan gebruikt worden om de gezagsrelaties van alle bewoners van een adres (verblijfsobject) te krijgen.
+
+Je kan gezag vragen van een minderjarige. Dan krijg je als antwoord wie er gezag heeft over deze minderjarige. Bijvoorbeeld de ouders van deze minderjarige.
+
+Je kan ook gezag vragen van een meerderjarige. Dan krijg je als antwoord over wie deze meerderjarige gezag heeft. Bijvoorbeeld over de kinderen van deze meerderjarige. Je krijgt dan alleen informatie over het gezag dat deze persoon wel heeft. Er wordt geen informatie geleverd over kinderen van de persoon waar deze geen gezag (meer) over heeft.
+
+Gezag geeft in de API een lijst. Hierin kunnen 0, 1 of meerdere gezagvoorkomens geleverd worden.
+
+Wanneer je het gezag vraagt van een minderjarige heeft het antwoord altijd exact 1 item gezag, tenzij de minderjarige is overleden.
+
+Wanneer je het gezag vraagt van een meerderjarige kan het antwoord 0 (de persoon heeft over niemand gezag), 1 of meerdere items krijgen. Bijvoorbeeld wanneer de persoon gezag heeft over meerdere kinderen. Elk gezagvoorkomen (elk item in de lijst gezag) geeft dan het gezag over één minderjarige.
+
+Elk geleverd gezag kan één de volgende types hebben
+- TweehoofdigOuderlijkGezag
+- EenhoofdigOuderlijkGezag
+- GezamenlijkGezag
+- Voogdij
+- TijdelijkGeenGezag
+- GezagNietTeBepalen
+
+### TweehoofdigOuderlijkGezag
+
```

Comment:
Kunnen we onderstaande niet beter beschrijven in feature vorm met de twee operaties zoekMetAdresseerbaarobjectIdentificatie en raadpleegMetBurgerservicenummer? Bij de laatste kunnen we misschien ook een voorbeeld geven van een vraag waarbij meerder burgerservicenummers worden opgegeven.